upgrade/1.2/Stage_2.md

```diff
@@ -43,6 +43,10 @@ For `ncn-m001`, use `ncn-m002` as the stable NCN. Use `bond0.cmn0`/CAN IP addres
    ncn-m002# mkdir -p /etc/cray/upgrade/csm/${CSM_RELEASE}
 
    ncn-m002# scp ncn-m001:/etc/cray/upgrade/csm/myenv /etc/cray/upgrade/csm/myenv
+   
+   ncn-m002# scp ncn-m001:/root/output.log /root/pre-m001-reboot-upgrade.log
+   
+   ncn-m002# cray artifacts create config-data pre-m001-reboot-upgrade.log /root/output.log
```

Comment:
Is /root/output.log supposed to be the file on m002?   Looks like that should be /root/pre-m001-reboot-upgrade.log.

Reply:
Corrected